vendor/locales/custom-en.yml

```
@@ -52,6 +52,13 @@ en:
     <summary class="welcome">Pre-requirements</summary>
 
     <ol>
+    <li><span>Create an Azure Service Principal. This can be done either using the Azure portal or the azure command line tool.</span>
```

Small typo and rewording:
```suggestion
    <li><span>Create an Azure Service Principal. This can be done either by using the Azure portal or the `az` CLI.</span>
```

---

Changed. Thanks!